src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -29,7 +29,7 @@ azure-mgmt-cognitiveservices==6.3.0
 azure-mgmt-compute==19.0.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
-azure-mgmt-containerregistry==3.0.0rc16
+azure-mgmt-containerregistry==3.0.0rc17
```

Comment:
since SDK version changed, I'll suppose you've already run live tests :)

Reply:
@yungezz There is no api version bump in this version. The spec change PR is here FYI: https://github.com/Azure/azure-rest-api-specs/pull/13001
But I still ran a few more tests live to provide enough confidence for this PR. Please take a look and let me know if you have any other suggestions.